push/templates/push/details.html

```diff
@@ -0,0 +1,19 @@
+{% extends "dashboard/base.html" %}
+
+{% block content %}
+    {% if not request.user.is_authenticated %}
+        {% include 'dashboard/includes/authenticate.html' %}
```

Comment:
Rather than include the authenticate template at this URL, we should wrap the views in login_required to bounce the user to the login page and bring them back here. Can be filed as a follow-up issue/TODO.